### ISSUE TITLE
Docker Compose Version change and setting Network during Build process

### ISSUE THREAD

**Issue by g7morris** [Islandora-Collaboration-Group/ISLE#95]
Please investigate the `network` command explained here:

https://docs.docker.com/compose/compose-file/compose-file-v2/#network

If updating the current v2 (only) `docker-compose.yml` file to v2.2 & use of syntax from the example given, would allow for **Apache site installation** during build

`Set the network containers will connect to for the RUN instructions during build.

build:
  context: .
  network: host

build:
  context: .
  network: custom_network_1`

**Comment by br2490** [comment]
+1 

Questions:
- [ ] Why isn't docker-compose respecting the build context's `network` setting?
- [ ] Is trying with just `docker build` a step in figuring this out, as it says you can pass the --net= arg on the CLI?
- [ ] How can we bring up a container during the `build` on a specific `docker network` so that stack components dependent can communicate?

Link provided by @g7morris is important first steps to try and figure this one out; assuming it is even possible without getting outside the box (e.g. proxying or routing) and so far out of scope it's not worth effort during Alpha stages)

tl;dr:  Docker: respect build network specified when you're building a services stack.  Especially if they're clearly `depends_on:` and therefore required during build?  right.

**Comment by br2490** [comment]
I think this ticket may be closed as it is superseded by a number of tickets which expand on this issue more in depth and (some) are solved as we're going.  I'm not sure if this ticket needs to sit queued.

@g7morris thoughts? if you concur would you do the honor?
